integration/test_aggregate_load_text_vector.py

```
@@ -0,0 +1,99 @@
+        self._create_index(client, key_type)
+
+        result = client.execute_command(
+            "FT.AGGREGATE", "idx", "@price:[-inf +inf]",
+            "LOAD", "1", "@title",
+        )
+
+        expected_field = b"title" if key_type == "HASH" else b"$.title"
+
+        assert result[0] == 3
+        for row in result[1:]:
+            fields = dict(zip(row[::2], row[1::2]))
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)